pkgs/by-name/in/invidious-router/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGo122Module, fetchFromGitLab }:
+let
+  pname = "invidious-router";
+  version = "1.0";
+in
+buildGo122Module {
+  inherit pname version;
+
+  src = fetchFromGitLab {
+    owner = "gaincoder";
+    repo = "invidious-router";
+    rev = version;
+    hash = "sha256-6apw+UnhDSuPa9QB3S8b8Ej3NJTK+UCdbDjC2LkhLIg=";
+  };
+
+  vendorHash = "sha256-c03vYidm8SkoesRVQZdg/bCp9LIpdTmpXdfwInlHBKk=";
+
+  doCheck = true;
+
+  meta = {
+    homepage = "https://gitlab.com/gaincoder/invidious-router/";
+    description = "A Go application that routes requests to different Invidious instances based on their health status and (optional) response time";
+    license = with lib.licenses; [ mit ];
```

Comment:
```suggestion
    license = lib.licenses.mit;
```

Reply:
@AndersonTorres told me to do so, I initally had this specified like you propose. I don't want to change this as it is already approved.